minijinja/src/filters.rs

```diff
@@ -406,7 +406,11 @@ mod builtins {
             Ok(Value::from(s.chars().rev().collect::<String>()))
         } else if matches!(v.kind(), ValueKind::Seq) {
             Ok(Value::from(
-                ok!(v.as_slice()).iter().rev().cloned().collect::<Vec<_>>(),
+                ok!(v.as_cow_slice())
+                    .iter()
+                    .rev()
+                    .cloned()
+                    .collect::<Vec<_>>(),
```

Comment:
This can now be implemented really cheaply via a custom `AsSeq`. I suspect there are many other such filters that can now also be optimized.